### ISSUE TITLE
Notification/Confirmation Boxes

### ISSUE THREAD

**Issue by michaelcullum** [phpBB-Blog/phpBB-Blog-for-3.1#51]
For confirmation boxes (Are you sure you would like to XYZ? or The XYZ has been successfully submitted) phpBB will now be using http://www.erichynds.com/examples/jquery-notify/ (Source: https://github.com/ehynds/jquery-notify )

The blog should also use it.

It would be nice to use AJAX for submission and to check for new comments and blog posts and update the post but thats something for later on.

**Comment by imkingdavid** [comment]
I agree with this. But we need to program this first without ajax and then add the ajax on top.

**Comment by michaelcullum** [comment]
or base our test boards on AJAX + events combined.
